org.apache.directory.studio.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.apache.directory.studio
+
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+
+command: ApacheDirectoryStudio
+finish-args:
+  - --device=dri
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --persist=.ApacheDirectoryStudio
+  - --persist=.eclipse
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
```

Comment:
FWIW while the app does *work* with Wayland, it's definitely buggier than with X11. For instance, the "are you sure you want to quit?" dialog is unusably small when first opened.

However I think it's probably good enough.